packages/cozy-client/src/store/queries.js

```diff
@@ -15,6 +15,13 @@ import flag from 'cozy-flags'
 import { getDocumentFromSlice } from './documents'
 import { isReceivingMutationResult } from './mutations'
 import { properId } from './helpers'
+import { isAGetByIdQuery, QueryDefinition } from '../queries/dsl'
+import { QueryState  } from '../types'
+
+/**
+ * @typedef {object} InitQueryOptions
```

Comment:
where is it used? (todo check if it's present in next commits)

Reply:
I've re-ordered the commits and think this is fixed.